res/skins/LateNight/waveform_and_mixer.xml

```diff
@@ -157,7 +157,7 @@
 					</Template>
 					<Template src="skin:waveform.xml">
 						<SetVariable name="channum">1</SetVariable>
-                        <SetVariable name="deck_color">#EECE33</SetVariable>
+                        <SetVariable name="deck_color">#E7C413</SetVariable>
```

Comment:
no it shouldn't be changed elsewhere... eece33 is "latenight yellow" and should be used in most places.  This particular variable is used to set the value of SignalColor in waveform.xml, so it's misleading that it's called deck_color.